external/YamlCppLib.cmake

```diff
@@ -39,10 +39,15 @@ set(YAML_CPP_INSTALL OFF CACHE BOOL "prevent install, not needed." FORCE)
 set(YAML_CPP_BUILD_TOOLS OFF CACHE BOOL "prevent tools from being build" FORCE) 
 set(YAML_CPP_BUILD_TESTS OFF CACHE BOOL "prevent gtest from being build" FORCE) 
 set(YAML_CPP_BUILD_CONTRIB OFF CACHE BOOL "prevent contrib modules" FORCE) 
+set(MSVC_SHARED_RT OFF CACHE BOOL "Use MT rather than MD" FORCE)
```

Comment:
nit, comment what is MT / MD?

Reply:
> comment what is MT / MD?

Compile option, /MT is static multi-threaded, and /MD is dynamic multi-threaded.  The /MD is kind of like -fPIC on Linux, it allows the code to be added to a shared library i.e. a DLL. (which is what a .pyd is).

I wanted to use /MT for the static libraries but it turns out I might have to switch to /MD.  Code in a library of any kind must all be the same flavor.  I will see if it is even possible to build a static library containing /MD code.